mtp_api/apps/account/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+from transaction.models import Transaction
+
+
+class FileType(models.Model):
```

Comment:
i suppose `FileType` could have a text code as the primary key so that it's not auto-increment dependent? i'm just not totally sure it warrants a whole model/serialiser/view/url collection…